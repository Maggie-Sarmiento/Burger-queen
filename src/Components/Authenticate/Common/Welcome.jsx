/* eslint-disable react/prop-types */
import React from 'react';
import styles from './Welcome.module.css';

const Welcome = () => (
  <section className={styles.welcome}>
    <div className={styles.txt}>
      <p className={styles.hello}>Hola!</p>
      <p className={styles.message}>
        Selecciona cualquier sección del menú
        lateral para empezar a tomar ordenes,
        servir ordenes, administrar al personal y
        menú según tu rol asignado.
      </p>
    </div>
    <div className={styles.images}>
      <img src="/yellowspot.jpg" alt="spot" className={styles.spot} />
      <img src="/waiter.jpg" alt="spot" className={styles.waiter} />
    </div>
  </section>
);

export default Welcome;
