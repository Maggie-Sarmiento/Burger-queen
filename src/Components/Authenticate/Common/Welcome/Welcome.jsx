/* eslint-disable react/prop-types */
import React from 'react';
import styles from './Welcome.module.css';

const Welcome = () => (
  <section className={styles.welcome}>
    <div className={styles.txt}>
      <p className={styles.hello}>Bienvenidx a Burger Queen!</p>
      <p className={styles.message}>
        En esta App, podras encontrar las herramientas para poder llevar
        a cabo tus tareas en Burguer Queen.
      </p>
      <p className={styles.message}>
        Selecciona cualquier sección del menú
        lateral para empezar a tomar/servir ordenes, administrar al personal y
        el menú según tu rol asignado.
      </p>
      <p>
        Si tienes dudas o preguntas sobre el manejo de la App
        Marca al 01-800-BQMX
      </p>
    </div>
    <div className={styles.images}>
      <img src="/yellowspot.jpg" alt="spot" className={styles.spot} />
      <img src="/waiter.jpg" alt="spot" className={styles.waiter} />
    </div>
  </section>
);

export default Welcome;
