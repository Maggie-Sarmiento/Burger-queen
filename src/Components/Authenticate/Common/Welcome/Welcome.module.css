@import url('https://fonts.googleapis.com/css2?family=Manrope:wght@500;600;700;800&family=Montserrat:wght@200;300;500&display=swap');

.welcome{
  margin-top: 10%;
  display: flex;
  flex-direction: column;
  height: 80vh;
}

.txt{
  width: 92%;
  padding-left: 1em;
  padding-right: 1em;
  font-family: 'Montserrat', sans-serif;
}

.hello {
  margin-top: 2em;
  font-size: 2em;
  font-weight: bolder;
  color: #F3B240;
}

.message {
  width: 105%;
  margin-bottom: 2em;
}

.images {
  position: relative;
}

.spot {
  position: absolute;
  width: 110%;
}

.waiter {
  top: 5em;
  right: 20%;
  position: absolute;
  border-radius: 50%;
  width: 70%;
}

@media (min-width: 550px) {
  .welcome{
    margin-top: 30%;
    height: 85vh;
  }

  .hello{
    margin-top: 1em;
  }

  .spot {
    position: absolute;
    width: 70%;
    top: 10%;
  }

  .waiter {
    top: 5em;
    right: 20%;
    width: 60%;
  }
}

@media (min-width: 1000px) {
  .welcome{
    margin-top: 5%;
    flex-direction: row;
    height: 70vh;
  }

  .hello {
    font-size: 2.4em;
  }

  .txt{
    width: 40%;
  }

  .spot {
    position: fixed;
  }

  .waiter {
    top: 7em;
    right: 5%;
    width: 80%;
  }

  .images{
    width: 60%;
  }

  .txt{
    padding-left: 1.5em;
  }
}

@media (min-width: 1200px) {
  .welcome{
    margin-top: 8.1%;
    height: 75vh;
  }

  .waiter {
    top: 0;
    right: 5%;
    width: 75%;
  }
}

@media (min-width: 1400px) {
  .welcome{
    margin-top: 10%;
  }

  .waiter {
    top: 3em;
    right: 5%;
    width: 80%;
  }
}

@media (min-width: 1900px) {
  .welcome{
    margin-top: 10%;
  }

  .txt {
    width: 50%;
    margin-left: 3em;
  }

  .images{
    width: 60%;
  }

  .hello{
    font-size: 2.5em;
  }

  .message{
    margin-bottom: 2.5em;
  }

  .waiter {
    top: 3em;
    right: 5%;
    width: 65%;
  }
}