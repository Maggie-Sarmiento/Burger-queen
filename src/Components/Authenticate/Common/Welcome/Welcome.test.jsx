/* eslint-disable no-undef */
import React from 'react';
import { create, act } from 'react-test-renderer';
import Welcome from './Welcome';

describe('test Welcome', () => {
  test('render component welcome', async () => {
    let root;
    act(() => {
      root = create(
        <Welcome />,
      );
    });
    expect(root.toJSON()).toMatchSnapshot();
  });
});
