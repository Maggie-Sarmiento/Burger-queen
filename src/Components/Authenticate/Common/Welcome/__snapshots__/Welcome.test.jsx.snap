// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Welcome render component welcome 1`] = `
<section
  className="welcome"
>
  <div
    className="txt"
  >
    <p
      className="hello"
    >
      Bienvenidx a Burger Queen!
    </p>
    <p
      className="message"
    >
      En esta App, podras encontrar las herramientas para poder llevar a cabo tus tareas en Burguer Queen.
    </p>
    <p
      className="message"
    >
      Selecciona cualquier sección del menú lateral para empezar a tomar/servir ordenes, administrar al personal y el menú según tu rol asignado.
    </p>
    <p>
      Si tienes dudas o preguntas sobre el manejo de la App Marca al 01-800-BQMX
    </p>
  </div>
  <div
    className="images"
  >
    <img
      alt="spot"
      className="spot"
      src="/yellowspot.jpg"
    />
    <img
      alt="spot"
      className="waiter"
      src="/waiter.jpg"
    />
  </div>
</section>
`;
